shared/styles.js

```diff
@@ -6,11 +6,10 @@ export const globalStyles = (
     styles={css`
       html,
       body {
-        padding: 3rem 1rem;
+        padding: 0;
```

Comment:
I think all these styles should be extracted to `index.css`, but maybe there's something legacy that I'm missing?

Reply:
this could be moved over to `index.css`: 
```
 .bmpui-ui-watermark {
        display: none !important;
}
```
and the rest of `shared/styles.js` removed. Ideally I'd like to avoid transferring any styles from current site.

Reply:
This file can go, Tailwind take cares of defaults and [tailwindcss-typography's](https://github.com/tailwindlabs/tailwindcss-typography) `prose` classes are great for simple typesetting.